lightning/mydump/loader.go

```diff
@@ -76,10 +78,20 @@ type mdLoaderSetup struct {
 }
 
 func NewMyDumpLoader(cfg *config.Config) (*MDLoader, error) {
+	var r *router.Table
+	if len(cfg.Routes) > 0 {
+		var err error
+		r, err = router.NewTableRouter(false, cfg.Routes)
```

Comment:
do we need to support case-sensitive route by adding a config item (like `case-sensitive`)?

Reply:
While we could add such config, I'd like to know what's the reason supporting this, since the identifiers are always case-insensitive.

Reply:
identifiers in MySQL are case-sensitive. In DM, we meet some cases where identifiers like `DB.TBL` and `db.tbl` both exist in one instance.

Reply:
recommended that all tools be unified.